cmd/kube-apiserver/app/server.go

```diff
@@ -84,8 +84,10 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/auth/authenticator/token/bootstrap"
 )
 
-const etcdRetryLimit = 60
-const etcdRetryInterval = 1 * time.Second
+const (
+	etcdRetryLimit    = 60
```

Comment:
what's the point?

Reply:
Instead of using `const` every time we define a constant, we can define it directly in the constant block.

Reply:
/cc @yue9944882